integration-test/src/main/java/com/sequenceiq/it/cloudbreak/actor/CloudbreakUser.java

```diff
@@ -12,19 +12,30 @@
 
     private String displayName;
 
+    private Boolean admin;
```

Comment:
can't we just use the primitive? Constructors could be simpler also.

Reply:
sure